src/player.py

```diff
@@ -1,25 +1,39 @@
+from src.game import Game
+
+
 # inherits from game class
 # this is because we need the players to access methods such as the score
 class Player(Game):
     # initialize hand with games deck
-    def __init__(self, game):
+    def __init__(self):
+        print("I compile")
 
     # lay down, should just call players hand easily
+    # put down a full set, or add to a set that is already laid down
     def lay_down(self):
+        print("I compile")
 
     # decides whether to go out
     def go_out(self):
+        print("I compile")
 
-    # obvious
+    # put down one card to discard pile
     def discard(self):
+        print("I compile")
 
     # pickup from deck
     def pick_up(self):
+        print("I compile")
 
     # pickup from discard deck
     def pick_up_discards(self):
+        print("I compile")
```

Comment:
you need to pass params such as deck into this

Reply:
is deck not public?

Reply:
it is not